docs/hybrid-slurm-cluster/blueprints/static-cluster.yaml

```diff
@@ -15,13 +15,32 @@
 ---
 
 blueprint_name: static-slurm-cluster
+toolkit_modules_url: github.com/GoogleCloudPlatform/cluster-toolkit
+toolkit_modules_version: v1.41.0
 
 vars:
   project_id:  ## <<PROJECT_A>>
   deployment_name: cluster
   region: us-central1
   zone: us-central1-c
 
+terraform_providers:
+  google:
+    source: hashicorp/google
+    version: 5.45.0
+    configuration:
+      project: $(vars.project_id)
```

Comment:
nit. do we have do it? I've though that toolkit would substitute those configs automagically.